packages/docusaurus-theme-classic/src/theme/TagsListByLetter/index.tsx

```diff
@@ -6,7 +6,9 @@
  */
 
 import React from 'react';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import {listTagsByLetters, type TagLetterEntry} from '@docusaurus/theme-common';
+import {type CharMap} from '@docusaurus/types/src/i18n';
```

Comment:
Don't deep-import. Export this type from the main interface. Also, we use `import type {}` wherever applicable, so this should be something like `import type {CharMap} from '@docusaurus/types';` after you've exported it from the types package.

Reply:
nice advice, and solved now ~